### PR TITLE
Get release 3.0 ready

### PR DESCRIPTION
**Pull request by JakeLin**
After months hard work, I think it about the time to ship 🎉🎉🎉

@tbaranes @lastMove can you guys have to look and let me know if you find something is missing for release 3?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,15 +153,15 @@ Copy and paste `IBAnimatable` folder in your Xcode project.
...
+Add `.Package(url: "https://github.com/IBanimatable/IBanimatable.git", majorVersion: 3)` to your `Package.swift`
 
 ### [CocoaPods](https://cocoapods.org)
 
 Add `pod 'IBAnimatable'` to your Podfile.
 
 ### [Carthage](https://github.com/Carthage/Carthage)
 
-Add `github "IBanimatable/IBAnimatable" ~> 2` to your Cartfile.
+Add `github "IBanimatable/IBAnimatable" ~> 3` to your Cartfile.
```
Comment (tbaranes) [inline README.md]: You can just put `github "IBanimatable/IBAnimatable"` in your Cartfile, it's not mandatory to specify the version in the README. That will make each new release easier for you

**Review by tbaranes — APPROVED** [review]
Let's do it 🎉

**Comment by JakeLin** [comment]
@lastMove I saw you have created some branches, please let me know if you think we are ready to release version 3, thanks.

**Comment by JakeLin** [comment]
@tbaranes Have addressed your comment, thanks.

**Comment by lastMove** [comment]
@JakeLin it's OK you can publish it. 🎉 🎈 😎

**Comment by JakeLin** [comment]
Thanks, will do it tonight 🚢🚢🚢

**Pull request merged**